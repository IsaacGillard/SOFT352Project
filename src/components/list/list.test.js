import React from 'react';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import ListPage from './ListPage';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

test('ListPage component should render as expected', () => {
  const component = shallow(<ListPage />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
