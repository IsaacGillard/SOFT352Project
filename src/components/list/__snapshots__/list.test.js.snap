// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListPage component should render as expected 1`] = `
<div
  className="row"
>
  <div
    className="col-12 col-sm-12"
  >
    <h2>
      â€˜s Profile
    </h2>
    <h3>
      Artists They Like
    </h3>
    <div />
    <h3>
      Albums They Own
    </h3>
    <div />
  </div>
</div>
`;
